draft-ietf-rats-architecture.md

```diff
@@ -1432,6 +1434,15 @@ the meantime the Attester has been compromised.
 Reordering and dropping attacks are mitigated if the transport provides the ability to detect reordering and drop.
 However, the delay attack described above can't be thwarted in this manner.
 
+## Trust Anchor Protection
+
+As noted in {{trustmodel}}, Verifiers and Relying Parties have trust anchor stores
+that must be secured.  Specifically, a Trust Anchor Store must resist
+modification against unauthorized insertion, deletion, and modification.
+
+If certificates are used as trust anchors, Verifiers and Relying Parties are also
+responsible for validating the entire certificate chain up to the trust anchor,
+which includes checking for certificate revocation.
```

Comment:
maybe we could reference the RFC(s) that are normative for certificate chain validation here?

Reply:
i.e., [Section 6 of RFC5280](https://tools.ietf.org/html/rfc5280#section-6)

Reply:
Done